seed/models/data_quality.py

```diff
@@ -1042,12 +1089,13 @@ def update_status_label(self, label_class, rule, linked_id, row_id):
                     raise IntegrityError(
                         'Label with super_organization_id={} cannot be applied to a record with parent '
                         'organization_id={}.'.format(
+                            label_org_id,
```

Comment:
Why are contents of this error message being changed?
```suggestion
```

Reply:
No recall on this one. Must have been 'command + s' turning into 'command + d'.